extensions/amp-lightbox-gallery/1.0/base-element.js

```diff
@@ -58,31 +58,18 @@ export class BaseElement extends PreactBaseElement {
     });
   }
 
-  /** @override */
-  unmountCallback() {
-    this.removeAsContainer();
-  }
-
   /** @private */
   beforeOpen_() {
```

Comment:
Since these are overridden, they should be `@protected`. You'll need to remove the underscore as well.

Reply:
good point, done!